### PR TITLE
Update portuguese translation

### PR DESCRIPTION
**Pull request by yuripourre**
Portuguese (BR) translation 100% complete.

### CODE REVIEW

**Comment by AJenbo** [comment]
@julealgon or @altiereslima do one of you have time to review this?

File: Translations/pt_BR.po [hunk 1]
```diff
@@ -718,78 +685,103 @@ msgstr ""
...
 msgstr "Sim"
 
-#: Source/DiabloUI/selyesno.cpp:55 Source/stores.cpp:979
+#: Source/DiabloUI/selyesno.cpp:55 Source/stores.cpp:1005
 msgid "No"
 msgstr "Não"
 
+#: Source/DiabloUI/settingsmenu.cpp:300
+msgid "Bound key:"
+msgstr "Associar tecla:"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Wouldn't this be preferred here?
```suggestion
msgstr "Tecla associada:"
```

File: Translations/pt_BR.po [hunk 2]
```diff
@@ -718,78 +685,103 @@ msgstr ""
...
+msgstr "Menu anterior"
+
 #: Source/DiabloUI/support_lines.cpp:8
 msgid ""
 "We maintain a chat server at Discord.gg/YQKCAYQ Follow the links to join our "
 "community where we talk about things related to Diablo, and the Hellfire "
 "expansion."
 msgstr ""
+"Nós mantemos um servidor de chat em Discord.gg/YQKCAYQ Clique nos links para "
+"se juntar a comunidade onde nós conversamos sobre assuntos relacionadis ao "
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
"se juntar a comunidade onde nós conversamos sobre assuntos relacionados ao "
```

File: Translations/pt_BR.po [hunk 3]
```diff
@@ -828,8 +820,12 @@ msgid ""
 "CC-BY 4.0. The port also makes use of SDL which is licensed under the zlib-"
 "license. See the ReadMe for further details."
 msgstr ""
+"\tEste port faz uso das fontes CharisSILB, Unifont, e Noto que são "
+"licenciadas sob a SIL Open Font License, e também a fonte Twitmoji que é "
+"licensiada sob CC-BY 4.0. O port também faz uso da SDL, a qual é licenciada "
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
"licenciada sob CC-BY 4.0. O port também faz uso da SDL, a qual é licenciada "
```

File: Translations/pt_BR.po [hunk 4]
```diff
@@ -875,359 +871,306 @@ msgstr ""
...
+msgid "Player attack"
+msgstr "Hostil a jogadores"
+
+#: Source/control.cpp:667
+msgid "Hotkey: {:s}"
+msgstr "Tecla de atalho: {:s}"
+
+#: Source/control.cpp:675
+msgid "Select current spell button"
+msgstr "Selecionar botão do feitiço atual"
```
Comment (julealgon) [inline Translations/pt_BR.po]: This one feels incorrect to me. I believe the direct translation should be:
```suggestion
msgstr "Botão para selecionar feitiço atual"
```
However, that obviously looks terrible. I think we should seriously reconsider the original text here and remove the "button" part altogether, then translate as
```suggestion
msgstr "Selecionar feitiço atual"
```

File: Translations/pt_BR.po [hunk 5]
```diff
@@ -875,359 +871,306 @@ msgstr ""
...
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:794
+#: Source/diablo.cpp:799
 msgid ""
 "\n"
 "Game selection:\n"
 msgstr ""
+"\n"
+"Selecionar jogo:\n"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
"Seleção do jogo:\n"
```
Having said that, I'd also change the original text on this one to match your translation ("Select game").

File: Translations/pt_BR.po [hunk 6]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
 msgstr "enquanto estiver em lojas"
 
-#. TRANSLATORS: {:s} means: Character Name, Game Version, Game Difficulty.
+#: Source/diablo.cpp:1390
+msgid "Belt item {}"
+msgstr "Item do cinto {}"
+
+#: Source/diablo.cpp:1391
+msgid "Use Belt item."
+msgstr "Usar item do cinto."
```
Comment (julealgon) [inline Translations/pt_BR.po]: For consistency?
```suggestion
msgstr "Usar item do Cinto."
```

File: Translations/pt_BR.po [hunk 7]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+msgid "Quick spell {}"
+msgstr "Feitiço rápido {}"
+
+#: Source/diablo.cpp:1406
+msgid "Hotkey for skill or spell."
+msgstr "Tecla de atalho para habilidade ou feitiço."
+
+#: Source/diablo.cpp:1423
+msgid "Speedbook"
+msgstr "Livro de feitiços"
```
Comment (julealgon) [inline Translations/pt_BR.po]: This is incorrect. This is the translation for "spellbook", not "speedbook".
```suggestion
msgstr "Livro de acesso rápido"
```

File: Translations/pt_BR.po [hunk 8]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+
 #: Source/diablo.cpp:1451
-msgid "{:s}, version = {:s}, mode = {:s}"
-msgstr "{:s}, versão = {:s}, modo = {:s}"
+msgid "Stops walking and cancel pending actions."
+msgstr "Para de andar e cancela ações pendentes"
+
+#: Source/diablo.cpp:1459
+msgid "Open Inventory screen."
+msgstr "Abre tela de inventário"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Again consistency:
```suggestion
msgstr "Abre tela de Inventário"
```

File: Translations/pt_BR.po [hunk 9]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+
+#: Source/diablo.cpp:1465
+msgid "Character"
+msgstr "Person."
+
+#: Source/diablo.cpp:1466
+msgid "Open Character screen."
+msgstr "Abre tela de personagem"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Abre tela de Personagem"
```

File: Translations/pt_BR.po [hunk 10]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+
+#: Source/diablo.cpp:1472
+msgid "Quest log"
+msgstr "Registro de missões"
+
+#: Source/diablo.cpp:1473
+msgid "Open Quest log."
+msgstr "Abre registro de missões."
```
Comment (julealgon) [inline Translations/pt_BR.po]: I'd go with a different, more natural IMHO, tense here:
```suggestion
msgstr "Abrir registro de missões."
```

File: Translations/pt_BR.po [hunk 11]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+
+#: Source/diablo.cpp:1479
+msgid "Spellbook"
+msgstr "Livro de feitiços"
+
+#: Source/diablo.cpp:1480
+msgid "Open Spellbook."
+msgstr "Abre livro de feitiços."
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Abrir livro de feitiços."
```

File: Translations/pt_BR.po [hunk 12]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+msgid "Zoom Game Screen."
+msgstr "Zoom da tela do jogo"
+
+#: Source/diablo.cpp:1506
+msgid "Decrease Gamma"
+msgstr "Diminui Gama"
+
+#: Source/diablo.cpp:1507
+msgid "Reduce screen brightness."
+msgstr "Reduz o brilho da tela."
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Reduzir o brilho da tela."
```

File: Translations/pt_BR.po [hunk 13]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+msgid "Zoom"
+msgstr "Zoom"
+
+#: Source/diablo.cpp:1497
+msgid "Zoom Game Screen."
+msgstr "Zoom da tela do jogo"
+
+#: Source/diablo.cpp:1506
+msgid "Decrease Gamma"
+msgstr "Diminui Gama"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Diminuir Gama"
```

File: Translations/pt_BR.po [hunk 14]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+
+#: Source/diablo.cpp:1507
+msgid "Reduce screen brightness."
+msgstr "Reduz o brilho da tela."
+
+#: Source/diablo.cpp:1513
+msgid "Increase Gamma"
+msgstr "Aumenta Gama"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Aumentar Gama"
```

File: Translations/pt_BR.po [hunk 15]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+
+#: Source/diablo.cpp:1514
+msgid "Increase screen brightness."
+msgstr "Aumenta o brilho da tela."
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Aumentar o brilho da tela."
```

File: Translations/pt_BR.po [hunk 16]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+msgid "Help"
+msgstr "Ajuda"
+
+#: Source/diablo.cpp:1521
+msgid "Open Help Screen."
+msgstr "Abre Tela de Ajuda."
+
+#: Source/diablo.cpp:1527
+msgid "Game info"
+msgstr "Informação do jogo"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Subjective, but I think this is better:
```suggestion
msgstr "Informações do jogo"
```

File: Translations/pt_BR.po [hunk 17]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+msgid "Cathedral"
+msgstr "Catedral"
+
+#: Source/discord/discord.cpp:66
+msgid "Catacombs"
+msgstr "Catacumbas"
+
+#: Source/discord/discord.cpp:66
+msgid "Caves"
+msgstr "Caverna"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Should we pluralize this as well? Seems consistent to me:
```suggestion
msgstr "Cavernas"
```

File: Translations/pt_BR.po [hunk 18]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+
+#. TRANSLATORS: Discord character, i.e. "Lv 6 Warrior"
+#: Source/discord/discord.cpp:88
+msgid "Lv {} {}"
+msgstr "Nvl {} {}"
+
+#. TRANSLATORS: Discord state i.e. "Nightmare difficulty"
+#: Source/discord/discord.cpp:100
+msgid "{} difficulty"
+msgstr "{} dificuldade"
```
Comment (julealgon) [inline Translations/pt_BR.po]: We should invert this:
```suggestion
msgstr "Dificuldade {}"
```

File: Translations/pt_BR.po [hunk 19]
```diff
@@ -1236,46 +1179,218 @@ msgstr ""
...
+
+#. TRANSLATORS: Discord activity, not in game
+#: Source/discord/discord.cpp:180
+msgid "In Menu"
+msgstr "Menu Principal"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Shouldn't this be:
```suggestion
msgstr "No Menu"
```

File: Translations/pt_BR.po [hunk 20]
```diff
@@ -1776,13 +1885,11 @@ msgstr ""
 "lançar um feitiço preparado simplesmente clicando com o botão direito na "
 "área de jogo."
 
-#: Source/help.cpp:79
-#, fuzzy
-#| msgid "$Using the Speedbook for Spells"
+#: Source/help.cpp:80
 msgid "$Using the Speedbook for Spells:"
-msgstr "$Uso do Livro de acesso rápido para feitiços"
+msgstr "$Uso do Livro de acesso rápido para feitiços:"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Would suggest:
```suggestion
msgstr "$Usando o Livro de acesso rápido para feitiços:"
```

File: Translations/pt_BR.po [hunk 21]
```diff
@@ -1793,25 +1900,19 @@ msgstr ""
...
-"preparado"
+"preparado."
 
-#: Source/help.cpp:86
-#, fuzzy
-#| msgid "$Setting Spell Hotkeys"
+#: Source/help.cpp:87
 msgid "$Setting Spell Hotkeys:"
-msgstr "$Configuração de teclas de atalho para feitiços"
+msgstr "$Configuração de teclas de atalho para feitiços:"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "$Configurando teclas de atalho para feitiços:"
```

File: Translations/pt_BR.po [hunk 22]
```diff
@@ -1822,2542 +1923,2518 @@ msgstr ""
...
 msgid "The Defender"
 msgstr "O defensor"
 
-#: Source/itemdat.cpp:418
-#, fuzzy
-#| msgid "Gryphons Claw"
+#: Source/itemdat.cpp:455
 msgid "Gryphon's Claw"
-msgstr "Garra de grifo"
+msgstr "Garra de Grifo"
```
Comment (julealgon) [inline Translations/pt_BR.po]: I wonder if this wouldn't be more correct:
```suggestion
msgstr "Garra do Grifo"
```

File: Translations/pt_BR.po [hunk 23]
```diff
@@ -4434,314 +4507,219 @@ msgid "Bone Gasher"
...
 msgctxt "monster"
 msgid "Fiend"
-msgstr ""
-"Pela Luz, eu ouvi falar desse demônio vil. Houve muitos que carregavam as "
-"cicatrizes de sua ira em seus corpos quando os poucos sobreviventes do "
-"ataque liderado por Lázarus se arrastaram para fora da Catedral. Eu não sei "
-"o que ele usou para cortar suas vítimas, mas não pode ser algo deste mundo. "
-"Deixou feridas podres com doenças, e mesmo eu as achei quase impossíveis de "
-"tratar. Cuidado se planeja batalhar contra esse demônio..."
+msgstr "Fiend"
```
Comment (julealgon) [inline Translations/pt_BR.po]: This has a proper translation.
```suggestion
msgstr "Demônio"
```

File: Translations/pt_BR.po [hunk 24]
```diff
@@ -4434,314 +4507,219 @@ msgid "Bone Gasher"
...
 msgid "The Butcher"
 msgstr "O Açougueiro"
 
 #: Source/monstdat.cpp:72
-#, fuzzy
-#| msgid "Overlord"
 msgctxt "monster"
 msgid "Overlord"
-msgstr "Elmo do suserano"
+msgstr ""
```
Comment (julealgon) [inline Translations/pt_BR.po]: Is this a mistake? Missing translation?

File: Translations/pt_BR.po [hunk 25]
```diff
@@ -4751,481 +4729,331 @@ msgid "Litch Demon"
...
-msgstr "Fogo "
+msgstr "Fogo da Perdição"
 
 #: Source/monstdat.cpp:95
-#, fuzzy
-#| msgid "Hell Burner"
 msgctxt "monster"
 msgid "Hell Burner"
-msgstr "Descer para o inferno"
+msgstr "Quimador do Inferno"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Queimador do Inferno"
```

File: Translations/pt_BR.po [hunk 26]
```diff
@@ -4751,481 +4729,331 @@ msgid "Litch Demon"
...
-msgstr "Ápice da tempestade"
+msgstr "Tempestade Vermelha"
 
 #: Source/monstdat.cpp:97
-#, fuzzy
-#| msgid "Storm Rider"
 msgctxt "monster"
 msgid "Storm Rider"
-msgstr "Ápice da tempestade"
+msgstr "Piloto da Tempestade"
```
Comment (julealgon) [inline Translations/pt_BR.po]: This looks terrible... I don't have a suggestion though.

File: Translations/pt_BR.po [hunk 27]
```diff
@@ -4751,481 +4729,331 @@ msgid "Litch Demon"
...
 msgid "Gargoyle"
 msgstr "Gárgula"
 
 #: Source/monstdat.cpp:103
-#, fuzzy
-#| msgid "Blood Claw"
 msgctxt "monster"
 msgid "Blood Claw"
-msgstr "do sangue"
+msgstr "Pata Sangrenta"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Claw is not "pata":
```suggestion
msgstr "Garra Sangrenta"
```

File: Translations/pt_BR.po [hunk 28]
```diff
@@ -4751,481 +4729,331 @@ msgid "Litch Demon"
...
-"outra chance de livrar o mundo de seu mal!"
+msgstr "O Lorde Sombrio"
 
 #: Source/monstdat.cpp:131
-#, fuzzy
-#| msgid "The Arch-Litch Malignus"
 msgctxt "monster"
 msgid "The Arch-Litch Malignus"
-msgstr "de arcanjo"
+msgstr "The Arch-Litch Malignus"
```
Comment (julealgon) [inline Translations/pt_BR.po]: This one _should_ have a translation.... "The" is missing,  "arch" is "arce", but I don't know if "lich"/"litch" exists...

Maybe we should just go with:
```suggestion
msgstr "O Arce-Litch Malignus"
```
?

File: Translations/pt_BR.po [hunk 29]
```diff
@@ -4751,481 +4729,331 @@ msgid "Litch Demon"
...
-msgstr "A câmara de osso"
+msgstr "Demônio de ossos"
 
 #: Source/monstdat.cpp:153
-#, fuzzy
-#| msgid "Arch Lich"
 msgctxt "monster"
 msgid "Arch Lich"
-msgstr "de arcanjo"
+msgstr "Arch Lich"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Same here:
```suggestion
msgstr "Arce Lich"
```

File: Translations/pt_BR.po [hunk 30]
```diff
@@ -5442,158 +5222,116 @@ msgid "Blackash the Burning"
...
 msgid "Shadowcrow"
 msgstr "Corvo das Sombras"
 
 #: Source/monstdat.cpp:509
-#, fuzzy
-#| msgid "Blightstone the Weak"
 msgctxt "monster"
 msgid "Blightstone the Weak"
-msgstr "Gharbad, o Fraco"
+msgstr "Blightstone, o Fraco"
```
Comment (julealgon) [inline Translations/pt_BR.po]: "Blight" and "stone" have translations. We can translate this one like the others.

File: Translations/pt_BR.po [hunk 31]
```diff
@@ -5442,158 +5222,116 @@ msgid "Blackash the Burning"
...
-msgstr "Rasga-Carne Sombra da Morte"
+msgstr "Sombra da Morte Rasga-Carne"
 
 #: Source/monstdat.cpp:518
-#, fuzzy
-#| msgid "Warmaggot the Mad"
 msgctxt "monster"
 msgid "Warmaggot the Mad"
-msgstr "Zhar, o Louco"
+msgstr "Warmaggot, o Louco"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Same here with "war" and "maggot"

File: Translations/pt_BR.po [hunk 32]
```diff
@@ -5834,22 +5530,16 @@ msgid "Sir Gorash"
...
 msgid "The Vizier"
 msgstr "O Vizir"
 
 #: Source/monstdat.cpp:567
-#, fuzzy
-#| msgid "Sapphire"
 msgctxt "monster"
 msgid "Zamphir"
-msgstr "safira"
+msgstr ""
```
Comment (julealgon) [inline Translations/pt_BR.po]: Wrong?

File: Translations/pt_BR.po [hunk 33]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+#: Source/objects.cpp:284
 msgid "A Spellbook"
 msgstr "Um livro de feitiços"
 
-#: Source/objects.cpp:5339
-msgid "Crucified Skeleton"
-msgstr "Esqueleto crucificado"
+#: Source/objects.cpp:5386
+msgid "Crucified Skeleton"
+msgstr "Esqueleto crucificado"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Casing consistency:
```suggestion
msgstr "Esqueleto Crucificado"
```

File: Translations/pt_BR.po [hunk 34]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5390
+msgid "Lever"
+msgstr "Alavanca"
+
+#: Source/objects.cpp:5399
+msgid "Open Door"
+msgstr "Porta aberta"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Porta Aberta"
```

File: Translations/pt_BR.po [hunk 35]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5401
+msgid "Closed Door"
+msgstr "Porta fechada"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Porta Fechada"
```

File: Translations/pt_BR.po [hunk 36]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5403
+msgid "Blocked Door"
+msgstr "Porta bloqueada"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Porta Bloqueada"
```

File: Translations/pt_BR.po [hunk 37]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5408
+msgid "Ancient Tome"
+msgstr "Tomo antigo"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Tomo Antigo"
```

File: Translations/pt_BR.po [hunk 38]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5410
+msgid "Book of Vileness"
+msgstr "Livro da vileza"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Livro da Vileza"
```

File: Translations/pt_BR.po [hunk 39]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5415
+msgid "Skull Lever"
+msgstr "Alavanca de caveira"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Alavanca de Caveira"
```

File: Translations/pt_BR.po [hunk 40]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5418
+msgid "Mythical Book"
+msgstr "Livro mítico"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Livro Mítico"
```

File: Translations/pt_BR.po [hunk 41]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5422
+msgid "Small Chest"
+msgstr "Baú pequeno"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Baú Pequeno"
```

File: Translations/pt_BR.po [hunk 42]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5426
+msgid "Chest"
+msgstr "Baú"
+
+#: Source/objects.cpp:5431
+msgid "Large Chest"
+msgstr "Baú grande"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Baú Grande"
```

File: Translations/pt_BR.po [hunk 43]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+msgstr "Barril"
+
+#. TRANSLATORS: {:s} will be a name from the Shrine block above
+#: Source/objects.cpp:5454
+msgid "{:s} Shrine"
+msgstr "Altar {:s}"
+
+#: Source/objects.cpp:5458
+msgid "Skeleton Tome"
+msgstr "Tomo do esqueleto"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Tomo do Esqueleto"
```

File: Translations/pt_BR.po [hunk 44]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/objects.cpp:5461
+msgid "Library Book"
+msgstr "Livro da biblioteca"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Livro da Biblioteca"
```

File: Translations/pt_BR.po [hunk 45]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+msgstr "Suporte Vil"
+
+#: Source/objects.cpp:5516
+msgid "Slain Hero"
+msgstr "Herói assassinado"
+
+#. TRANSLATORS: {:s} will either be a chest or a door
+#: Source/objects.cpp:5523
+msgid "Trapped {:s}"
+msgstr "{:s} aprisionado"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Poor translation I think:
```suggestion
msgstr "{:s} com armadilha"
```

File: Translations/pt_BR.po [hunk 46]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+msgid "OFF"
+msgstr ""
+
+#: Source/options.cpp:560
+msgid "Start Up"
+msgstr ""
+
+#: Source/options.cpp:560
+msgid "Start Up Settings"
+msgstr ""
```
Comment (julealgon) [inline Translations/pt_BR.po]: Missing?

File: Translations/pt_BR.po [hunk 47]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/options.cpp:567
+msgid "Restrict to Shareware"
+msgstr "Restringir ao Shareware"
+
+#: Source/options.cpp:567
+msgid ""
+"Makes the game compatible with the demo. Enables multiplayer with friends "
+"who don't own a full copy of Diablo."
+msgstr ""
```
Comment (julealgon) [inline Translations/pt_BR.po]: ?

File: Translations/pt_BR.po [hunk 48]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+
+#: Source/options.cpp:568 Source/options.cpp:574
+msgid "Intro"
+msgstr "Introdução"
+
+#: Source/options.cpp:568 Source/options.cpp:574
+msgid "Shown Intro cinematic."
+msgstr ""
```
Comment (julealgon) [inline Translations/pt_BR.po]: ?

File: Translations/pt_BR.po [hunk 49]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+msgid "Number of output channels."
+msgstr "Número de canais de saída"
+
+#: Source/options.cpp:626
+msgid "Buffer Size"
+msgstr "Tamanho do Buffer"
+
+#: Source/options.cpp:626
+msgid "Buffer size (number of frames per channel)."
+msgstr "Tamanho do Buffer (número de frames por canal)."
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Tamanho do Buffer (número de quadros por canal)."
```

File: Translations/pt_BR.po [hunk 50]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+msgid "Fullscreen"
+msgstr "Tela cheia"
+
+#: Source/options.cpp:745
+msgid "Display the game in windowed or fullscreen mode."
+msgstr ""
+
+#: Source/options.cpp:747
+msgid "Fit to Screen"
+msgstr "Caber na tela"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Would suggest:
```suggestion
msgstr "Ajustar à tela"
```

File: Translations/pt_BR.po [hunk 51]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+msgid "Color Cycling"
+msgstr ""
+
+#: Source/options.cpp:760
+msgid "Color cycling effect used for water, lava, and acid animation."
+msgstr ""
+
+#: Source/options.cpp:762
+msgid "Hardware Cursor"
+msgstr "Cursor do Hardware"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Cursor em Hardware"
```

File: Translations/pt_BR.po [hunk 52]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+msgid ""
+"Allow arrow/spell damage between players in multiplayer even when the "
+"friendly mode is on."
+msgstr ""
+"Permite causar danos por flechas/feitiços entre jogadores no multiplayer "
+"mesmo quando o modo amistoso está habilitado"
+
+#: Source/options.cpp:818
+msgid "Test Bard"
+msgstr "Bardo"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Why omit the "Test " prefix?

File: Translations/pt_BR.po [hunk 53]
```diff
@@ -5992,443 +5676,1016 @@ msgid "Player '{:s}' (level {:d}) is already in the game"
...
+msgstr ""
 
-#: Source/objects.cpp:5399
-msgid "Pod"
-msgstr "Ovo"
+#: Source/options.cpp:835
+#, fuzzy
+#| msgid "quick attack"
+msgid "Quick Cast"
+msgstr "ataque breve"
```
Comment (julealgon) [inline Translations/pt_BR.po]: Incorrect:
```suggestion
msgstr "conjuração breve"
```
(or something like that)

File: Translations/pt_BR.po [hunk 54]
```diff
@@ -6466,29 +6723,86 @@ msgstr "feitiços"
...
+msgstr "Nível {:d}"
+
+#: Source/panels/spell_book.cpp:164
+msgid "Unusable"
+msgstr "Inutilizável"
+
+#. TRANSLATORS: UI constrains, keep short please.
+#: Source/panels/spell_book.cpp:172
+msgid "Heals: {:d} - {:d}"
+msgstr "Curas: {:d} - {:d}"
```
Comment (julealgon) [inline Translations/pt_BR.po]:
```suggestion
msgstr "Cura: {:d} - {:d}"
```

**Review by julealgon — CHANGES_REQUESTED** [review]
Stopped commenting on casing inconsistencies and missing translation since there we just too many.

File: Translations/pt_BR.po (hunk 24 above)
Comment (yuripourre) [reply Translations/pt_BR.po]: Actually, this is probably done by an automatic action, it borrowed the translation from "OverLord's Helm" and pasted it here.

File: Translations/pt_BR.po (hunk 26 above)
Comment (yuripourre) [reply Translations/pt_BR.po]: Hahah, yeah. I am trying to think about names following a similar pattern but sometimes is really hard.

File: Translations/pt_BR.po (hunk 27 above)
Comment (yuripourre) [reply Translations/pt_BR.po]: Oops

File: Translations/pt_BR.po (hunk 45 above)
Comment (yuripourre) [reply Translations/pt_BR.po]: Boa!

**Comment by yuripourre** [comment]
100% Complete but a lot of conflicts. Bad timming :/

**Comment by glebm** [comment]
You could "update from source" in poedit and then use your version for all the conflicts.

**Comment by yuripourre** [comment]
@glebm thanks, I will try that.

**Comment by yuripourre** [comment]
I messed up with the rebase but now it seems fixed.

**Comment by AJenbo** [comment]
@altiereslima @yuripourre if either of you have time to update the translation now would be a good time to make it for the next release :)

There are about 320 lines that needs an update.

**Comment by AJenbo** [comment]
Ok, that sound perfect :) I realize that we aren't giving translators the working conditions here with it being done directly on GitHub and not having a formal string freeze process so really appreciate the effort :+1:

**Comment by AJenbo** [comment]
There's going to be and update to DevilutionX soon so now would be a good time to update the translation if you have time :)

**Pull request merged**